### ISSUE TITLE
Multi-line figure caption alignment is inconsistent.

### ISSUE THREAD

**Issue by newAM** [sphinx-contrib/confluencebuilder#283]
In 1.1 to 1.2 the default alignment figure directive changed:
https://jdknight.atlassian.net/wiki/spaces/confluencebuilder/pages/905773094/v1.2-images
https://jdknight.atlassian.net/wiki/spaces/confluencebuilder/pages/813170965/v1.1-images

With multiline captioned figures I used to see:
```
[[image]                             ]
[[line one]                          ]
[[line two]                          ]
```

Now I see:
```
[               [image]              ]
[               [line one]           ]
[[line two]                          ]
```

All lines after the first are left aligned.

This is from this RST:

```rst
.. figure: xyz.jpg

	line one

	line two
```

**Comment by jdknight** [comment]
As of Sphinx v2.0+, the [default alignment for tables and figures](https://github.com/sphinx-doc/sphinx/commit/25027945f59fc810b511374d9cf2fad67068ecdf) is of a `center` alignment. This extension takes this into consideration by applying Sphinx's default alignment based off the detected Sphinx version; however, the main issue is now that "line two" is not following the expected default alignment in this case -- which needs to be corrected.

Two other notables related to this:

1. While applying a `left` alignment (as in the validation example provided), it uses Confluence's `ac:align` attribute which will in turn float-left the element. With Sphinx stock themes, this does not happen. Maybe this extension should instead use not `ac:align` attribute when a `left` attribute is assigned to ensure content is left-align but not floated. Doing so would also prevent users who may wish to left-float-align images though.

2. Maybe it would be ideal to add a configuration option to allow a user to override the default alignment to apply. For example, maybe the introduction of a `confluence_default_alignment` option?

**Comment by newAM** [comment]
1. In this case my opinion would be that it is better to follow the behavior of sphinx stock themes, maybe this could be exposed by another config option?
2. A `confluence_default_alignment` would be handy for me!

**Comment by jdknight** [comment]
See also #290.

**Comment by jdknight** [comment]
A solution (outlined in #290) has been applied to `master` and will be made available on PyPI next stable release (v1.3+).

Default alignments should now properly work for figure legend elements. Users should be able to also use `confluence_default_alignment` to override default alignment values (for Sphinx 2.1+). Users can also still take advantage of float-left images (outside of figures), if desired (for the time being).

@newAM, thanks for bringing up this issue -- and hopefully the new configuration option does what you desire. This issue will now be closed; however, if there are other concerns not addresses, please feel free to bring them up.
